ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/participant/state/kvutils/KVUtilsPackageSpec.scala

```diff
@@ -63,6 +63,19 @@ class KVUtilsPackageSpec extends WordSpec with Matchers with BazelRunfiles {
       }
     }
 
+    "be able to pre-execute model-test.dar" in KVTest.runTest {
+      for {
+        // NOTE(JM): 'runTest' always uploads 'simpleArchive' by default.
+        preExecutionResult <- preExecuteArchives(
+          "model-test-submission",
+          testStablePackages.all: _*)
+          .map(_._2)
```

Comment:
As there are multiple assertions on the same result, for improving readability you could declare the relevant log entry here, i.e.:
```scala
for {
  preExecutionResult <- ...
  actualLogEntry = preExecutionResult.successfulLogEntry
} yield {
  actualLogEntry.getPayLoadCase shouldEqual ...
  ...
}
```

Reply:
Done.